Makefile

```diff
@@ -59,6 +60,24 @@ remote-garden-up:
 remote-garden-down:
 	@./hack/local-development/remote-garden/stop.sh $(REMOTE_GARDEN_LABEL)
 
+.PHONY: start-all
+start-all:
+	# start a screen session and open all servers in named tabs
+	echo "starting screen session with all servers"
+	@screen -ls $(DEV_SCREEN_NAME) || screen -AdmS $(DEV_SCREEN_NAME) -t tab0 bash
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t apiserver bash -c "make --debug=j start-apiserver; exec bash"
+	@./hack/wait-for-apiserver.sh
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t controller bash -c "make --debug=j start-controller-manager; exec bash"
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t scheduler bash -c "make --debug=j start-scheduler; exec bash"
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t gardenlet bash -c "make --debug=j start-gardenlet; exec bash"
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t admission bash -c "make --debug=j start-admission-controller; exec bash"
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t seed-admission bash -c "make --debug=j start-seed-admission-controller; exec bash"
```

Comment:
I guess this line should be removed as we would not start the seed-admission-controller (GSAC) locally (usually, at least not in conjunction with the others) because it's being deployed by the gardenlet and runs in the seed as a container.
For local development, the GSAC would be started locally, of course, but I guess it's not necessary in this "start-all"/screen session here.